app/controllers/concerns/localized.rb

```diff
@@ -26,6 +26,6 @@ def set_locale
   end
 
   def default_locale
-    ENV.fetch('DEFAULT_LOCALE') { I18n.default_locale }
+    ENV.fetch('DEFAULT_LOCALE') { http_accept_language.compatible_language_from(I18n.available_locales) }
```

Comment:
What happens when there is no accept-language header at all? I don't see a fallback to `I18n.default_locale` in the block.